src/App.js

```diff
@@ -1,24 +1,31 @@
-import logo from './logo.svg';
+import {
+  BrowserRouter, Routes, Route, Outlet,
+} from 'react-router-dom';
 import './App.css';
+import Navbar from './compoents/Navbar';
+import Book from './compoents/Book';
+import Categories from './compoents/Categories';
+
+function Layout() {
+  return (
+    <>
+      <Navbar />
+      <Outlet />
+    </>
+  );
+}
```

Comment:
- Good job done so far, I suggest having this component in another js file, so as to make your code as modular as possible 👍 👍 
- I also advise using arrow functions in order to maintain `ES6` syntax as it is the more accepted syntax in use globally. Please fix this 👍 👍